runtime/doc/dev_vimpatch.txt

```diff
@@ -184,6 +184,7 @@ information.
   mch_memmove                                             memmove
   vim_memset copy_chars copy_spaces                       memset
   vim_strbyte                                             strchr
+  vim_strnchr                                            strnchr
```

Comment:
Is `strnchr()`  really needed? https://github.com/neovim/neovim/pull/30526#discussion_r1939508781